dev-python/ptpython/ptpython-2.0.6.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
```

Comment:
Please drop py2.7 support from all. Gentoo is beginning the process of removing py2 support